DispaSET/preprocessing/preprocessing.py

```diff
@@ -154,24 +154,38 @@ def build_simulation(config):
 
     # Fuel prices:
     fuels = ['PriceOfNuclear', 'PriceOfBlackCoal', 'PriceOfGas', 'PriceOfFuelOil', 'PriceOfBiomass', 'PriceOfCO2', 'PriceOfLignite', 'PriceOfPeat']
-    FuelPrices = pd.DataFrame(columns=fuels, index=idx_utc_noloc)
-    for fuel in fuels:
-        if os.path.isfile(config[fuel]):
-            tmp = load_csv(config[fuel], header=None, index_col=0, parse_dates=True)
-            FuelPrices[fuel] = tmp[1][idx_utc_noloc].values
-        elif isinstance(config['default'][fuel], (int, float, complex)):
-            logging.warning('No data file found for "' + fuel + '. Using default value ' + str(config['default'][fuel]) + ' EUR')
-            FuelPrices[fuel] = pd.Series(config['default'][fuel], index=idx_utc_noloc)
-        # Special case for lignite and peat, for backward compatibility
-        elif fuel == 'PriceOfLignite':
-            logging.warning('No price data found for "' + fuel + '. Using the same value as for Black Coal')
-            FuelPrices[fuel] = FuelPrices['PriceOfBlackCoal']
-        elif fuel == 'PriceOfPeat':
-            logging.warning('No price data found for "' + fuel + '. Using the same value as for biomass')
-            FuelPrices[fuel] = FuelPrices['PriceOfBiomass']
-        else:
-            logging.warning('No data file or default value found for "' + fuel + '. Assuming zero marginal price!')
-            FuelPrices[fuel] = pd.Series(0, index=idx_utc_noloc)
+    fuel_series = list(fuels*len(config['countries']))
+    zone_series = []
+    for zone in config['countries']:
+        zone_series = zone_series+[zone]*len(fuels)
+    zonefuels_array = [zone_series, fuel_series]
+    FuelPrices = pd.DataFrame(columns=zonefuels_array, index=idx_utc_noloc, dtype=float)
+    idx = pd.IndexSlice
+    tmp = pd.DataFrame()
+    for zone in config['countries']:
+        for fuel in fuels:
+            try:
+                tmp = load_csv(config[fuel], header=0, index_col=0, parse_dates=True)
+            except:
```

Comment:
If the file does not exist then the tmp var from the previous loop will be used in the following conditional checks.. That's an error